templates/registry/markdown/examples_macros.j2

```diff
@@ -1,4 +1,23 @@
-{% macro print_examples(examples) %}{%- for e in examples %}{%if loop.first == false %}; {% endif %}`{{ e | trim }}`{%- endfor %}{% endmacro %}
+{%- macro escape_multiline_markdown_cell(value) -%}
```

Comment:
(not blocking) Should this be a "filter" we add in jinja from weaver?

We should open a feature request for this, it seems useful.

Reply:
Makes sense, I can follow up on this - https://github.com/open-telemetry/weaver/issues/893